pkg/webhooks/clusterqueue_webhook.go

```diff
@@ -191,7 +155,6 @@ func validateFlavorQuotas(flavorQuotas kueue.FlavorQuotas, coveredResources []co
 		if rq.BorrowingLimit != nil {
 			borrowingLimitPath := path.Child("borrowingLimit")
 			allErrs = append(allErrs, validateResourceQuantity(*rq.BorrowingLimit, borrowingLimitPath)...)
-			allErrs = append(allErrs, validateLimit(*rq.BorrowingLimit, cohort, borrowingLimitPath)...)
 		}
 		if features.Enabled(features.LendingLimit) && rq.LendingLimit != nil {
 			lendingLimitPath := path.Child("lendingLimit")
```

Comment:
quantities can be validated via CEL too https://kubernetes.io/docs/reference/using-api/cel/#kubernetes-quantity-library

Reply:
According to this table https://kubernetes.io/docs/reference/using-api/cel/#cel-options-language-features-and-libraries quantity library requires Kubernetes 1.29 -although then, in the quantity subsection states 1.28-, is it safe to replace the quantity validations with CEL? Wouldn't introduce breaking changes for the supported versions earlier than 1.29?

Reply:
oh I see. I missed that.
Can you leave an open issue to change this when 1.28 reaches EoL?